src/modules/hip/kernel/resample.hpp

```diff
@@ -0,0 +1,298 @@
+#include <hip/hip_runtime.h>
+#include "rpp_hip_common.hpp"
+
+// -------------------- Set 0 - resample kernel device helpers  --------------------
+
+__device__ __forceinline__ float resample_hip_compute(float &x, float &scale, float &center, float *lookup, int &lookupSize)
+{
+    float locRaw = x * scale + center;
+    int locFloor = std::floor(locRaw);
+    float weight = locRaw - locFloor;
+    locFloor = std::max(std::min(locFloor, lookupSize - 2), 0);
```

Comment:
The vectorized and unvectorized versions of this function have a difference in use of min/max. Pls check if its needed and that we are doing the same op

Reply:
Yes
We dont need to do min max check for vectorized version as per the algorithm and needs to be checked only for Raw c code